test/nodes/core/function/10-function_spec.js

```diff
@@ -1718,9 +1718,13 @@ describe('function node', function() {
     describe("init function", function() {
 
         it('should delay handling messages until init completes', function(done) {
+            const timeout_ms = 200;
+            // Since helper.load uses process.nextTick timers might occasionally finish
+            // a couple of milliseconds too early, so give some leeway to the check.
+            const timeout_check_margin = 5;
             var flow = [{id:"n1",type:"function",wires:[["n2"]],initialize: `
                 return new Promise((resolve,reject) => {
-                    setTimeout(resolve,200)
+                    setTimeout(resolve, ${timeout_ms});
```

Comment:
I am keeping the variable inside `${...}` context since the function body is a string and I am quite certain that it will not capture any closure of outside variables (and even if it would work there is no benefit in doing so, it is better to expand the variable value at the time of assignment rather than to have a closure variable access later).